backend/src/main/java/com/google/minicrm/servlets/WebhookServlet.java

```diff
@@ -116,7 +119,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
     Lead newLead = Lead.fromReader(request.getReader(), advertiserKey);
 
-    //verify the lead
```

Comment:
Enhance your documentation here.

Reply:
see below:     //verify the lead - has correct google key for this advertiser. does it need more explanation?

Reply:
Actually yea I see that you already explained well on the comment before the dopost method so this should be enough.